livre/eq/zeq/communicator.cpp

```diff
@@ -259,7 +259,7 @@ class Communicator::Impl
         // Receiving all queued events from all receivers without blocking.
         for( auto subscriber : subscribers )
             while( subscriber->receive( 0 ))
-                /*nop*/ ;
+                _config.sendEvent( REDRAW );
```

Comment:
Should "have" been, yes. Was it ? I don't fully know the details ( I have only looked part of it on my scope of stories ). So, has to be tested.